torchbenchmark/models/torchrec_dlrm/metadata.yaml

```diff
@@ -0,0 +1,6 @@
+eval_benchmark: false
+eval_deterministic: false
+eval_nograd: true
+train_benchmark: false
+train_deterministic: false
+skip_cuda_memory_leak: true
```

Comment:
We noticed this model will "memory leak" (i.e., some CUDA memory are still in use after deleting the model instance). In the past, it happened on LSTM models which has cyclic references. Not sure if it is the same case  for torchrec_dlrm, any ideas? @colin2328

Reply:
strange, I haven't seen this before